presto-hive/src/main/java/io/prestosql/plugin/hive/HiveClientModule.java

```diff
@@ -142,4 +146,10 @@ public ExecutorService createCachingHiveMetastoreExecutor(HiveConnectorId hiveCl
     {
         return transactionHandle -> ((HiveMetadata) transactionManager.get(transactionHandle)).getMetastore();
     }
+
+    @Provides
+    List<DynamicConfigurationUpdater> getDynamicUpdaterList(Set<DynamicConfigurationUpdater> updaters)
```

Comment:
We can remove this method if we change `HiveHdfsConfiguration` to take a `Set` instead of a `List`. Using a `Set` seems more correct anyway, since there is no ordering to the collection.